Bot.rb

```diff
@@ -0,0 +1,103 @@
+require 'telegram/bot'
+require 'dotenv'
+require 'sqlite3'
+Dotenv.load
+$token = ENV["BOT_TOKEN"]
```

Comment:
you use global variables, in most cases, it's a bad practice.
Try to use yml file as source file for you story. You will be able to define as much levels as you need, including steps, making the program more robust and readable.

```ruby
require 'yaml'
story = YAML.load_file('config/story.yml')
```